resources/istio-kyma-patch/README.md

```diff
@@ -0,0 +1,14 @@
+# Istio Kyma patch
+
+## Overview
+
+This chart packs [patch script](../../components/istio-kyma-patch/README.md) as kubernetes job.
```

Comment:
``This chart packs the [patch script](../../components/istio-kyma-patch/README.md) as a Kubernetes job.``